pytrip/tripexecuter2/execute.py

```diff
@@ -0,0 +1,496 @@
+#
+#    Copyright (C) 2010-2017 PyTRiP98 Developers.
+#
+#    This file is part of PyTRiP98.
+#
+#    PyTRiP98 is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU General Public License as published by
+#    the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    PyTRiP98 is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU General Public License for more details.
+#
+#    You should have received a copy of the GNU General Public License
+#    along with PyTRiP98.  If not, see <http://www.gnu.org/licenses/>.
+#
+"""
+This file contains the Execute() class which can execute a Plan() object locally or on a remote server
+whith a complete TRiP98 installtions.
+"""
+import os
+import shutil
+import tarfile
+
+from subprocess import Popen, PIPE
+
+from pytrip.dos import DosCube
+from pytrip.let import LETCube
+from pytrip.ctx import CtxCube
+
+import uuid
+import logging
+
+logger = logging.getLogger(__name__)
+
+try:
+    import paramiko
+except:
+    logger.warning("Paramiko package not installed, only local TRiP access possible")
+
+
+class Execute():
+    def __init__(self, ctx, vdx):
+        """ Initialize the Execute class.
+        :params CtxCube() ctx: the CT images as a regular pytrip.CtxCube() object.
+        :params VdxCube() ctx: the contours as a regular pytrip.VdxCube() object.
+
+        If the Ctx and Vdx obejcts are constant, this object can be used to execute multiple
```

Comment:
Python doesn't have the concept of constants, so better to reformulate this comment